[2021-12-28 13:48:32,057] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_louth.scrap_data scheduled__2021-12-27T10:15:00+00:00 [queued]>
[2021-12-28 13:48:32,103] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_louth.scrap_data scheduled__2021-12-27T10:15:00+00:00 [queued]>
[2021-12-28 13:48:32,105] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:48:32,106] {taskinstance.py:1242} INFO - Starting attempt 4 of 4
[2021-12-28 13:48:32,108] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-28 13:48:32,137] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-27 10:15:00+00:00
[2021-12-28 13:48:32,165] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_louth', 'scrap_data', 'scheduled__2021-12-27T10:15:00+00:00', '--job-id', '5119', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_louth.py', '--cfg-path', '/tmp/tmpw2ppqbxd', '--error-file', '/tmp/tmp7jzjf50r']
[2021-12-28 13:48:32,167] {standard_task_runner.py:77} INFO - Job 5119: Subtask scrap_data
[2021-12-28 13:48:32,156] {standard_task_runner.py:52} INFO - Started process 56479 to run task
[2021-12-28 13:48:32,364] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_louth.scrap_data scheduled__2021-12-27T10:15:00+00:00 [running]> on host 0c125a5bba95
[2021-12-28 13:48:32,659] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_louth
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-27T10:15:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-27T10:15:00+00:00
[2021-12-28 13:48:32,694] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-28 13:48:32,698] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py louth
[2021-12-28 13:48:37,866] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-28 13:48:37,991] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-28 13:48:37,992] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-28 13:48:37,993] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-28 13:48:37,994] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-28 13:48:37,995] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-28 13:48:38,063] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-28 13:48:38,067] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-28 13:48:38,069] {spark_submit.py:514} INFO - deployMode              null
[2021-12-28 13:48:38,070] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-28 13:48:38,072] {spark_submit.py:514} INFO - executorCores           null
[2021-12-28 13:48:38,073] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-28 13:48:38,076] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-28 13:48:38,077] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-28 13:48:38,078] {spark_submit.py:514} INFO - driverCores             null
[2021-12-28 13:48:38,079] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-28 13:48:38,080] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-28 13:48:38,082] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-28 13:48:38,083] {spark_submit.py:514} INFO - supervise               false
[2021-12-28 13:48:38,085] {spark_submit.py:514} INFO - queue                   null
[2021-12-28 13:48:38,085] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-28 13:48:38,086] {spark_submit.py:514} INFO - files                   null
[2021-12-28 13:48:38,087] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-28 13:48:38,088] {spark_submit.py:514} INFO - archives                null
[2021-12-28 13:48:38,090] {spark_submit.py:514} INFO - mainClass               null
[2021-12-28 13:48:38,091] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 13:48:38,092] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-28 13:48:38,093] {spark_submit.py:514} INFO - childArgs               [louth]
[2021-12-28 13:48:38,093] {spark_submit.py:514} INFO - jars                    null
[2021-12-28 13:48:38,094] {spark_submit.py:514} INFO - packages                null
[2021-12-28 13:48:38,095] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-28 13:48:38,096] {spark_submit.py:514} INFO - repositories            null
[2021-12-28 13:48:38,097] {spark_submit.py:514} INFO - verbose                 true
[2021-12-28 13:48:38,098] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:38,099] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-28 13:48:38,100] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-28 13:48:38,101] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:48:38,102] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:38,103] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:38,480] {spark_submit.py:514} INFO - Main class:
[2021-12-28 13:48:38,481] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-28 13:48:38,482] {spark_submit.py:514} INFO - Arguments:
[2021-12-28 13:48:38,483] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-28 13:48:38,484] {spark_submit.py:514} INFO - null
[2021-12-28 13:48:38,485] {spark_submit.py:514} INFO - louth
[2021-12-28 13:48:38,485] {spark_submit.py:514} INFO - --verbose
[2021-12-28 13:48:38,486] {spark_submit.py:514} INFO - Spark config:
[2021-12-28 13:48:38,487] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-28 13:48:38,488] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-28 13:48:38,489] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-28 13:48:38,490] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-28 13:48:38,490] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-28 13:48:38,491] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:38,492] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:38,493] {spark_submit.py:514} INFO - 
[2021-12-28 13:48:39,929] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-28 13:48:39,986] {spark_submit.py:514} INFO - 21/12/28 13:48:39 INFO SparkContext: Running Spark version 3.2.0
[2021-12-28 13:48:40,142] {spark_submit.py:514} INFO - 21/12/28 13:48:40 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-28 13:48:40,367] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO ResourceUtils: ==============================================================
[2021-12-28 13:48:40,369] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-28 13:48:40,371] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO ResourceUtils: ==============================================================
[2021-12-28 13:48:40,373] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-28 13:48:40,412] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-28 13:48:40,437] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO ResourceProfile: Limiting resource is cpu
[2021-12-28 13:48:40,438] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-28 13:48:40,542] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO SecurityManager: Changing view acls to: ***
[2021-12-28 13:48:40,543] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO SecurityManager: Changing modify acls to: ***
[2021-12-28 13:48:40,544] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO SecurityManager: Changing view acls groups to:
[2021-12-28 13:48:40,545] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO SecurityManager: Changing modify acls groups to:
[2021-12-28 13:48:40,559] {spark_submit.py:514} INFO - 21/12/28 13:48:40 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-28 13:48:41,168] {spark_submit.py:514} INFO - 21/12/28 13:48:41 INFO Utils: Successfully started service 'sparkDriver' on port 38359.
[2021-12-28 13:48:41,297] {spark_submit.py:514} INFO - 21/12/28 13:48:41 INFO SparkEnv: Registering MapOutputTracker
[2021-12-28 13:48:41,433] {spark_submit.py:514} INFO - 21/12/28 13:48:41 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-28 13:48:41,484] {spark_submit.py:514} INFO - 21/12/28 13:48:41 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-28 13:48:41,485] {spark_submit.py:514} INFO - 21/12/28 13:48:41 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-28 13:48:41,496] {spark_submit.py:514} INFO - 21/12/28 13:48:41 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-28 13:48:41,547] {spark_submit.py:514} INFO - 21/12/28 13:48:41 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-4c820161-6c85-4d2d-9739-dd3009f82ba8
[2021-12-28 13:48:41,617] {spark_submit.py:514} INFO - 21/12/28 13:48:41 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-28 13:48:41,648] {spark_submit.py:514} INFO - 21/12/28 13:48:41 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-28 13:48:41,997] {spark_submit.py:514} INFO - 21/12/28 13:48:41 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-28 13:48:41,998] {spark_submit.py:514} INFO - 21/12/28 13:48:41 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2021-12-28 13:48:41,999] {spark_submit.py:514} INFO - 21/12/28 13:48:41 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2021-12-28 13:48:42,000] {spark_submit.py:514} INFO - 21/12/28 13:48:41 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2021-12-28 13:48:42,000] {spark_submit.py:514} INFO - 21/12/28 13:48:41 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2021-12-28 13:48:42,002] {spark_submit.py:514} INFO - 21/12/28 13:48:41 WARN Utils: Service 'SparkUI' could not bind on port 4045. Attempting port 4046.
[2021-12-28 13:48:42,024] {spark_submit.py:514} INFO - 21/12/28 13:48:42 INFO Utils: Successfully started service 'SparkUI' on port 4046.
[2021-12-28 13:48:42,153] {spark_submit.py:514} INFO - 21/12/28 13:48:42 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://0c125a5bba95:4046
[2021-12-28 13:48:42,516] {spark_submit.py:514} INFO - 21/12/28 13:48:42 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:48:42,640] {spark_submit.py:514} INFO - 21/12/28 13:48:42 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:48:42,641] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:48:42,642] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:48:42,643] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:48:42,643] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:48:42,644] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:48:42,645] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:48:42,646] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:48:42,647] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:48:42,648] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:48:42,648] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:48:42,649] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:48:42,650] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:48:42,650] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:48:42,651] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:48:42,652] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:48:42,653] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:48:42,653] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:48:42,654] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:48:42,655] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:48:42,656] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:48:42,656] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:48:42,657] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:48:42,658] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:48:42,658] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:48:42,659] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:48:42,660] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:48:42,660] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:48:42,661] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:48:42,662] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:48:42,663] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:48:42,664] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:48:42,665] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:48:42,666] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:48:42,667] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:48:42,668] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:48:42,668] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:48:42,669] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:48:42,670] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:48:42,670] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:48:42,671] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:48:42,672] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:48:42,672] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:48:42,673] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:48:42,674] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:48:42,674] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:48:42,675] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:48:42,675] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:48:42,676] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:48:42,676] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:48:42,677] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:48:42,678] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:48:42,678] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:48:42,679] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:48:42,679] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:48:42,680] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:48:42,681] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:48:42,681] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:49:02,515] {spark_submit.py:514} INFO - 21/12/28 13:49:02 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:49:02,529] {spark_submit.py:514} INFO - 21/12/28 13:49:02 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:49:02,529] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:49:02,530] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:49:02,531] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:49:02,532] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:49:02,533] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:49:02,534] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:49:02,535] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:49:02,535] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:49:02,536] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:49:02,536] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:49:02,537] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:49:02,538] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:49:02,538] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:49:02,539] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:49:02,540] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:49:02,540] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:49:02,541] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:49:02,542] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:49:02,542] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:49:02,543] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:49:02,543] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:49:02,544] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:49:02,545] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:49:02,546] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:49:02,546] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:49:02,547] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:49:02,547] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:49:02,548] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:49:02,549] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:49:02,549] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:49:02,550] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:49:02,551] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:49:02,551] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:49:02,552] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:49:02,553] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:49:02,554] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:49:02,554] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:49:02,555] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:49:02,555] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:49:02,556] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:49:02,557] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:49:02,557] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:49:02,558] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:49:02,558] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:49:02,559] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:49:02,560] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:49:02,560] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:49:02,561] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:49:02,562] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:49:02,562] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:49:02,563] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:49:02,564] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:49:02,564] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:49:02,565] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:49:02,566] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:49:02,566] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:49:02,567] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:49:22,481] {spark_submit.py:514} INFO - 21/12/28 13:49:22 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-28 13:49:22,498] {spark_submit.py:514} INFO - 21/12/28 13:49:22 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-28 13:49:22,499] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-28 13:49:22,500] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-28 13:49:22,501] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-28 13:49:22,501] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-28 13:49:22,502] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-28 13:49:22,503] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-28 13:49:22,505] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-28 13:49:22,505] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-28 13:49:22,506] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-28 13:49:22,507] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-28 13:49:22,508] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:49:22,508] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-28 13:49:22,509] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-28 13:49:22,510] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-28 13:49:22,511] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-28 13:49:22,512] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-28 13:49:22,512] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-28 13:49:22,513] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-28 13:49:22,514] {spark_submit.py:514} INFO - ... 4 more
[2021-12-28 13:49:22,515] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-28 13:49:22,516] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-28 13:49:22,517] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-28 13:49:22,517] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-28 13:49:22,518] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-28 13:49:22,519] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-28 13:49:22,520] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-28 13:49:22,521] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-28 13:49:22,522] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-28 13:49:22,522] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-28 13:49:22,523] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-28 13:49:22,524] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-28 13:49:22,525] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-28 13:49:22,526] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-28 13:49:22,526] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-28 13:49:22,527] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-28 13:49:22,528] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-28 13:49:22,528] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-28 13:49:22,530] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-28 13:49:22,531] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-28 13:49:22,531] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-28 13:49:22,532] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-28 13:49:22,533] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-28 13:49:22,534] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-28 13:49:22,534] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-28 13:49:22,535] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-28 13:49:22,536] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-28 13:49:22,537] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-28 13:49:22,537] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-28 13:49:22,538] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-28 13:49:22,539] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-28 13:49:22,539] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-28 13:49:22,540] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-28 13:49:22,541] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-28 13:49:22,541] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-28 13:49:22,542] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-28 13:49:22,543] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-28 13:49:22,543] {spark_submit.py:514} INFO - ... 1 more
[2021-12-28 13:49:42,450] {spark_submit.py:514} INFO - 21/12/28 13:49:42 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-28 13:49:42,458] {spark_submit.py:514} INFO - 21/12/28 13:49:42 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-28 13:49:42,472] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 41251.
[2021-12-28 13:49:42,473] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO NettyBlockTransferService: Server created on 0c125a5bba95:41251
[2021-12-28 13:49:42,476] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-28 13:49:42,500] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 0c125a5bba95, 41251, None)
[2021-12-28 13:49:42,534] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO BlockManagerMasterEndpoint: Registering block manager 0c125a5bba95:41251 with 434.4 MiB RAM, BlockManagerId(driver, 0c125a5bba95, 41251, None)
[2021-12-28 13:49:42,537] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO SparkUI: Stopped Spark web UI at http://0c125a5bba95:4046
[2021-12-28 13:49:42,544] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 0c125a5bba95, 41251, None)
[2021-12-28 13:49:42,546] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 0c125a5bba95, 41251, None)
[2021-12-28 13:49:42,560] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-28 13:49:42,575] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-28 13:49:42,604] {spark_submit.py:514} INFO - 21/12/28 13:49:42 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-28 13:49:42,661] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-28 13:49:42,731] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO MemoryStore: MemoryStore cleared
[2021-12-28 13:49:42,735] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO BlockManager: BlockManager stopped
[2021-12-28 13:49:42,756] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-28 13:49:42,769] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-28 13:49:42,855] {spark_submit.py:514} INFO - 21/12/28 13:49:42 INFO SparkContext: Successfully stopped SparkContext
[2021-12-28 13:49:43,022] {spark_submit.py:514} INFO - 21/12/28 13:49:43 ERROR SparkContext: Error initializing SparkContext.
[2021-12-28 13:49:43,023] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:49:43,024] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:49:43,026] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:49:43,027] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:49:43,028] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:49:43,029] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:49:43,032] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:49:43,035] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:49:43,037] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:49:43,038] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:49:43,039] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:49:43,040] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:49:43,041] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:49:43,042] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:49:43,043] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:49:43,045] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:49:43,046] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:49:43,048] {spark_submit.py:514} INFO - 21/12/28 13:49:43 INFO SparkContext: SparkContext already stopped.
[2021-12-28 13:49:43,051] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-28 13:49:43,053] {spark_submit.py:514} INFO - File "/opt/***/jobs/get_ads_per_county.py", line 13, in <module>
[2021-12-28 13:49:43,054] {spark_submit.py:514} INFO - .appName("Get Ads per county") \
[2021-12-28 13:49:43,055] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-28 13:49:43,057] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-28 13:49:43,058] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-28 13:49:43,059] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-28 13:49:43,060] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-28 13:49:43,064] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-28 13:49:43,066] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-28 13:49:43,067] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-28 13:49:43,069] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-28 13:49:43,071] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-28 13:49:43,071] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-28 13:49:43,072] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-28 13:49:43,074] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-28 13:49:43,075] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-28 13:49:43,076] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-28 13:49:43,078] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-28 13:49:43,081] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-28 13:49:43,087] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-28 13:49:43,088] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-28 13:49:43,091] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-28 13:49:43,095] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-28 13:49:43,096] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-28 13:49:43,097] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-28 13:49:43,098] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-28 13:49:43,099] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-28 13:49:43,100] {spark_submit.py:514} INFO - 
[2021-12-28 13:49:43,204] {spark_submit.py:514} INFO - 21/12/28 13:49:43 INFO ShutdownHookManager: Shutdown hook called
[2021-12-28 13:49:43,206] {spark_submit.py:514} INFO - 21/12/28 13:49:43 INFO ShutdownHookManager: Deleting directory /tmp/spark-725af1e4-b4a0-48c0-a111-e5d85c85615c
[2021-12-28 13:49:43,212] {spark_submit.py:514} INFO - 21/12/28 13:49:43 INFO ShutdownHookManager: Deleting directory /tmp/spark-da03fd4f-5f77-47e5-b624-00e305de7b38
[2021-12-28 13:49:43,456] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py louth. Error code is: 1.
[2021-12-28 13:49:43,472] {taskinstance.py:1280} INFO - Marking task as FAILED. dag_id=scraping_daft_louth, task_id=scrap_data, execution_date=20211227T101500, start_date=20211228T134832, end_date=20211228T134943
[2021-12-28 13:49:43,570] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/email.py:102: PendingDeprecationWarning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=smtp_mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2021-12-28 13:49:43,575] {email.py:208} INFO - Email alerting: attempt 1
[2021-12-28 13:49:44,354] {email.py:220} INFO - Sent an alert email to ['clement.liscoet@gmail.com']
[2021-12-28 13:49:45,259] {standard_task_runner.py:91} ERROR - Failed to execute job 5119 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py louth. Error code is: 1.
[2021-12-28 13:49:45,299] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-28 13:49:45,519] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
